README.md

```diff
@@ -138,6 +138,33 @@ Setup SQS for AGS Grade Pass Back
 2. The URI for a SQS queue should be filled in for either `lti13.grade-passback-queue` or as `LTI13_GRADE_PASSBACK_QUEUE` environment variable.
 3. The region of the SQS queue should be filled in for either `cloud.aws.region.static` in the application.properties file or `AWS_REGION` as an environment variable.
 
+Turning off AWS Integration for Local Development
+-------------------------------------------------
+Prior to running `mvn clean install spring-boot:run`, do the following:
+1. `export spring_profiles_active=no-aws`
+2. Ensure that the following property is present and uncommented in the application.properties file:
```

Comment:
Should we mention here and below, that if we are If we are creating a local `application-local.properties` file, these properties will need to be updated there instead?

Reply:
Good point - I've updated it to reference both files.